aspnet/getting-started/installing-on-mac.rst

```diff
@@ -26,13 +26,19 @@ You can also install ASP.NET 5 from the command-line. There are a few steps invo
 Install the .NET Version Manager (DNVM)
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Use the .NET Version Manager (DNVM) to install different versions of the .NET Execution Environment (DNX) on OS X.
+To install DNVM:
 
-To install DNVM run the following::
+#. Open a Terminal.
+#. Type ``cd ~/`` to go to your home folder.
+#. Enter ``touch .bash_profile`` to create a new bash profile.
```

Comment:
This is only necessary if you don't already have a bash profile and the install script will tell you to do this if you need to. Let's just leave the curl command.